operators/hack/run-e2e.sh

```diff
@@ -14,7 +14,7 @@ set -eu
 IMG="$1" # Docker image name
 TESTS_MATCH="$2" # Expression to match go test names (can be "")
 
-JOB_NAME="elastic-operators-e2e-tests"
```

Comment:
Is there room in the name (or should we make) for a random suffix? (example shows 6 chars, change the fold to shorten)

```
LC_CTYPE=C SUFFIX=$(tr -dc 'a-z0-9' < /dev/urandom | fold -w 6 | head -n 1)
```

If we're trying to avoid collisions, random suffix beats a datetime stamp? :)

Reply:
It looks like we don't need the datetime stamp, so perhaps just the random suffix is fine?

Reply:
yeah I think we'll be able to retrieve the date from the job resource if we want to.

Reply:
Fixed